.code-samples.meilisearch.yaml

```diff
@@ -1064,7 +1064,7 @@ update_experimental_features_1: |-
     -X PATCH 'http://localhost:7700/experimental-features/' \
     -H 'Content-Type: application/json'  \
     --data-binary '{
-      "scoreDetails": true
+      "exportPuffinReports": true
```

Comment:
Could we use `vectorStore` instead which is a way more interesting feature 😇

Reply:
As I mentioned in the postman PR, I sort of wanted to avoid emphasizing the connection of vector search with experimental features. My reasoning is that we often mention experimental features are unstable and recommend against using them in production environments, which is at odds with the rest of our communication being very much recommending people to use vector search as much as possible.

Happy to change this if you don't think it's a problem.

Reply:
Use `metrics` instead in this case. `exportPuffinReports` has almost zero feature interest for a classic user. They will use it when engine team will ask to use it in issue reports